package.json

```diff
@@ -51,5 +54,8 @@
   },
   "scripts": {
     "test": "grunt ci --verbose"
+  },
+  "browser": {
+    "pouchdb/lib/deps/request": "pouchdb/lib/deps/request-browser"
```

Comment:
if one of your peerDeps changes you make a new release. given the upstream uses semver.